terraform-plans/configs/github.tfvars

```diff
@@ -0,0 +1 @@
+owner = "Canonical"
```

Comment:
Compliant by CI: Maybe "canonical" with a lower case 'c'?

Reply:
I tried that localy, same. The app need to have read-only for organization/members.